build.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: "jellyfin-plugin-playbackreporting"
 guid: "5c534381-91a3-43cb-907a-35aa02eb9d2c"
-version: "4" # Please increment with each pull request
+version: "5" # Please increment with each pull request
 jellyfin_version: "10.4.0" # The earliest binary-compatible version
```

Comment:
did you change .net core version used in the build? 'cause if you didn't it won't work with 10.5+, and if you did you have to change this version as well

Reply:
sorry i've never done this before, could you explain what you mean please?

the only things i changed were the html pages so nothing major, am i supposed to update the .net core version somehow? I just used visual studio code. 

I built it for testing using "dotnet publish --configuration Release --output bin" 

Thanks

Reply:
This line should probably be updated: https://github.com/jellyfin/jellyfin-plugin-playbackreporting/blob/master/Jellyfin.Plugin.PlaybackReporting/Jellyfin.Plugin.PlaybackReporting.csproj#L3

paging @crobibero for help

Reply:
i had a go please check if correct. thanks